bpf/tc/attach.go

```diff
@@ -512,3 +549,39 @@ func (ap *AttachPoint) JumpMapFDMapKey() string {
 func (ap *AttachPoint) IfaceName() string {
 	return ap.Iface
 }
+
+func GetTCOpts(key string) (*libbpf.TCOpts, bool) {
+	optsLock.RLock()
+	defer optsLock.RUnlock()
+	val, ok := optsMap[key]
+	return val, ok
+}
+
+func InitTcOpts() {
```

Comment:
```suggestion
func InitTCOpts() {
```

I think you can just do vvv without the need of actually calling this function anywhere

`var optsMap = map[string]*libbpf.TCOpts{}`

https://play.golang.org/p/scOfvfYwxSG